src/App.tsx

```diff
@@ -4,12 +4,12 @@ import './App.css'
 import { SimpleStream } from './examples'
 
 function App() {
-  const { isLoading } = useSpawnEngine()
+  const { isLoading, port } = useSpawnEngine()
 
   return (
     <div className="App">
       <h1>Benchttp</h1>
-      {isLoading ? <div>Loading</div> : <SimpleStream />}
+      {isLoading ? <div>Loading</div> : <SimpleStream port={port} />}
```

Comment:
I though about it too when making the changes! But I went for the current implem because it was really easy to get it working.

Env variable seems to be a good candidate with low